examples/AlgorandDBStorage/AlgorandDBStorage.algo.ts

```diff
@@ -0,0 +1,66 @@
+import { Contract } from "@algorandfoundation/tealscript";
```

Comment:
This should be changed to `import { Contract } from '../../src/lib/index'`

Reply:
I wanted in this example to show independent way to implement it without the outer folders so that person can just copy it to custom folders and make it work. Dependency on parent folders does not make from it good example..

Also with the build.sh script i wanted to show how the build can be done in real world and client can be generated so that it can be used in cicd pipeline for example.

Reply:
Sorry just realized I missed your reply. Right now I want to keep the examples in this repo as importing from the current lib so they are guaranteed to be the latest version and can be used for testing. If you want to make a separate repo, I can make a "projects using TEALScript" section in the README.